docs/.vuepress/config.js

```diff
@@ -134,18 +134,17 @@ module.exports = {
                 ]
               },
               {
-                title: 'Examples & case studies',
+                title: 'Case studies',
                 sidebarDepth: 1,
                 collapsable: false,
                 children: [
-                  '/concepts/usage-ideas-examples',
-                  '/concepts/case-study-arbol',
-                  '/concepts/case-study-audius',
-                  '/concepts/case-study-fleek',
-                  '/concepts/case-study-iotex',
-                  '/concepts/case-study-likecoin',
-                  '/concepts/case-study-morpheus',
-                  '/concepts/case-study-openbazaar'
+                  ['/concepts/case-study-arbol', 'Arbol'],
+                  ['/concepts/case-study-audius', 'Audius'],
+                  ['/concepts/case-study-fleek', 'Fleek'],
+                  ['/concepts/case-study-iotex', 'Iotex'],
+                  ['/concepts/case-study-likecoin', 'LikeCoin'],
+                  ['/concepts/case-study-morpheus', 'Morpheus'],
```

Comment:
```suggestion
                  ['/concepts/case-study-morpheus', 'Morpheus.Network'],
```
(we'd originally left the network part out because it didn't fit in the menu!)